packages/cozy-stack-client/src/FileCollection.spec.js

```diff
@@ -480,7 +480,47 @@ describe('FileCollection', () => {
       })
     })
   })
+  describe('download', () => {
+    beforeEach(() => {
+      client.fetchJSON.mockResolvedValue({
+        data: {}
+      })
+    })
+
+    afterEach(() => {
+      client.fetchJSON.mockClear()
+    })
+
+    it('should use default file object', async () => {
```

Comment:
I don't understand the description, maybe "it should work when not specifying a revision ?"